### ISSUE TITLE
How do I add text to the steps?

### ISSUE THREAD

**Issue by hardikamal** [chenyun122/StepIndicator#7]
Hi @chenyun122 

Thanks for the awesome lib

How do I add text to the steps?

**Comment by chenyun122** [comment]
Do you mean the text under circle?

**Comment by hardikamal** [comment]
@chenyun122 yes, bottom in (left to right) or (right to left) and right/left side in (top to bottom) or (bottom to top)

**Comment by Kaakati** [comment]
Hello @chenyun122 

Having a text next to the circle in a vertical view or below or above in a horizontal view would be great.
I wanted to use your library but it didn't give me what i need so I ended up using this one untill probably someday, we'll have a text in your library.
https://github.com/yonat/StepProgressView

**Comment by Kaakati** [comment]
Hello @chenyun122 

It would be nice to have a UILabel either top or below circle in horizontal or left size or right side of circle in vertical, similar to this:

<img src="https://github.com/yonat/StepProgressView/raw/master/screenshots/blue.gif">

**Comment by creative-dev-lab** [comment]
Any update on this?

**Comment by india2sarthak** [comment]
I've come up with a very basic solution (https://github.com/chenyun122/StepIndicator/pull/29) to achieve this. 

Here's how you can use it..
```swift
// Initialize progressbar through code or storyboard
   let progressBar = StepIndicatorView()
// Set the data attributes
  progressBar.currentStep = 0
  progressBar.numberOfSteps = 3
// Set the desired titles
  progressBar.titles = ["One",  "Two", "Three"]
```
And this is how it looks ...

![Screenshot 2021-07-19 at 2 46 41 PM](https://user-images.githubusercontent.com/28748775/126135900-c9982d3c-a9bc-4fad-82bd-e56f350c2e15.png)
